sopel/modules/reddit.py

```diff
@@ -393,31 +401,31 @@ def get_channel_sfw(bot, trigger):
         bot.say('%s is flagged as NSFW' % channel)
 
 
-@require_chanmsg('.setspoilfree is only permitted in channels')
-@commands('setspoilerfree', 'setspoilfree')
-@example('.setspoilfree true')
-@example('.setspoilfree false')
+@plugin.require_chanmsg('.setspoilfree is only permitted in channels')
```

Comment:
Same issue as with `.setsfw`. Sigh.

```suggestion
@plugin.require_chanmsg('Only channels can be marked as spoiler-free.')
```